iris/sentryiris.go

```diff
@@ -59,8 +62,31 @@ func (h *handler) handle(ctx iris.Context) {
 		client.SetSDKIdentifier(sdkIdentifier)
 	}
 
+	options := []sentry.SpanOption{
+		sentry.WithOpName("http.server"),
+		sentry.ContinueFromRequest(ctx.Request()),
+		sentry.WithTransactionSource(sentry.SourceRoute),
+	}
+
+	currentRoute := ctx.GetCurrentRoute()
+
+	transaction := sentry.StartTransaction(
+		sentry.SetHubOnContext(ctx, hub),
+		fmt.Sprintf("%s %s", currentRoute.Method(), currentRoute.Path()),
+		options...,
+	)
+
+	defer func() {
+		hub.Scope().SetExtra("http.response.status_code", ctx.GetStatusCode())
+		transaction.Status = sentry.HTTPtoSpanStatus(ctx.GetStatusCode())
+		transaction.Finish()
+	}()
+
+	hub.Scope().SetExtra("http.request.method", ctx.Request().Method)
```

Comment:
Wait, why did I use `hub.Scope().SetExtra()` instead of `transaction.SetData()`, sigh...